src/components/Pipelines/PipelineDetail/PipelineDetail.js

```diff
@@ -59,37 +80,68 @@ const PipelineDetail = ({
                 </i>}
           </td>
         </tr>
+
         <tr>
-          <th>
+          <td className="text-center shrink-col em-padding-left em-padding-right">
+            <Icon icon="code" />
```

Comment:
Well, this one is used rather often. Maybe it is time to create a <CodeIcon> specialization.

Reply:
ok, will do